lib/dialects/postgres/query/pg-querycompiler.js

```diff
@@ -197,29 +197,12 @@ class QueryCompiler_PG extends QueryCompiler {
     }
   }
 
-  // Join array of table names and apply default schema.
-  _tableNames(tables) {
```

Comment:
When the `_tableNames` helper was written (#2835) there were multiple methods hitting it, but they've since been merged into the single `_lockingClause` method. Given that the only really interesting thing it was doing was prepending the schema name--which is the incorrect behavior this commit fixes--it didn't seem worth keeping at this point.